solr/solr-ref-guide/src/security-ui.adoc

```diff
@@ -0,0 +1,108 @@
+= Security UI
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+The Security screen allows administrators with the `security-edit` permission to manage users, roles, and permissions.
+The Security screen works with Solr running in cloud and standalone modes.
+
+.Security Screen
+image::images/solr-admin-ui/security.png[]
+
+== Getting Started
+
+The Security screen warns you if security is not enabled for Solr. You are strongly encouraged to enable security for Solr instances exposed on any network other than localhost.
+
+image::images/security-ui/security-not-enabled-warn.png[image,width=500]
+
+When first getting started with Solr, use the `bin/solr auth` command-line utility to enable security for your Solr installation (cloud mode only).
```

Comment:
Maybe add a link to `solr-control-script-reference.adoc#authentication` here (or somewhere in this section) so people can get the details on all the params if they need them?